stacks/signal-processing/jupyterhub.yaml

```diff
@@ -5,6 +5,8 @@ repo:
   url: https://jupyterhub.github.io/helm-chart/
 version: 2.0.0
 options:
+  labels:
+    stackable.tech/vendor: Stackable
```

Comment:
this throws an exception: "Additional property labels is not allowed"
```suggestion
```

Reply:
*Should* be fixed by 00adef6

Reply:
This doesn't seem to label anything on the pod but the error has gone.